src/stories/colour-tokens.js

```diff
@@ -0,0 +1,82 @@
+export default {
```

Comment:
This file is taken from the token build process from this work in progress PR
https://github.com/wellcometrust/design-system/pull/2

Reply:
Would an aim be to eventually remove this file and rely on design tokens from an external source?

Reply:
Yes this file is a stop-gap. The idea is to remove this and any similar build files from the codebase in favour of an npm process which uses tokens in JSON format to generate all outputs needed.

For more context, this file was generated by Style Dictionary using [config](https://github.com/wellcometrust/design-system/pull/2/files#diff-587cb980af76fdc7e52369fd0b9d926dff266976b6f8ac631e358fecc49ff8cf) and [colour/base.json](https://github.com/wellcometrust/design-system/pull/2/files#diff-703b1c57d8c36e3261c7a5a414e550f17ae43e611f6dfe00d7d1a01594f85ac2) from the afore mentioned PR.

Are you specifically talking about an external source from a product theming standpoint?

Reply:
Yep. Was concerned that we're defining tokens within the npm package itself (design-system), which seems like duplication (was imagining a single source of truth somewhere, probably a JSON file, which could be edited by designers to update values, which then permeate through the many apps which consume the design tokens).

Reply:
Yes, the end result will be the single source of truth you speak of